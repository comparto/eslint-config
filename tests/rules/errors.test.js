const errors = require('../../src/rules/errors')

test('errors rules', () => {
  expect(errors).toMatchSnapshot()
})
