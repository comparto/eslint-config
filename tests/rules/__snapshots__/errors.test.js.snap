// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`errors.js 1`] = `
Object {
  "rules": Object {
    "no-console": "warn",
  },
}
`;
